components/dashboard/src/App.tsx

```diff
@@ -19,6 +19,7 @@ import settingsMenu from './settings/settings-menu';
 const Account = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Account'));
 const Notifications = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Notifications'));
 const Plans = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Plans'));
+const Teams = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Teams'));
```

Comment:
**issue**: Not sure if this is related but loading the teams section takes a second or two! 🦦

Reply:
how do you measure? I'd need to reproduce somehow.

Reply:
Reloading Teams and any other page like Variables side-by-side.

| Teams | Variables l
|-|-|
| ![image](https://user-images.githubusercontent.com/120486/112979469-4aef8b00-9161-11eb-8084-0ece31f99e7f.png) | ![image](https://user-images.githubusercontent.com/120486/112979471-4c20b800-9161-11eb-8d2d-c940c11ec502.png) |

Reply:
But you've also seen the traffic on /teams, right? There is an external service which is loaded, etc. 
1. it shouldn't impact loading time of other pages
2. it should load in a reasonable time, where the time to render is significant 

Do you see a lagging first rendering of the content on /teams?

Reply:
> Do you see a lagging first rendering of the content on /teams?

I think, yes! Let's me screen capture this.

https://user-images.githubusercontent.com/120486/112984228-4fb73d80-9167-11eb-90ec-fbbf8385624a.mov

https://user-images.githubusercontent.com/120486/112984234-52199780-9167-11eb-8246-c49fade7bbcc.mov

Reply:
If you think this is more challenging to tackle here, let's leave this out of the scope of this PR and tackle separately and probably in an upcoming release! 🗺️

Reply:
<img width="1821" alt="Screen Shot 2021-03-30 at 15 38 58" src="https://user-images.githubusercontent.com/914497/112998173-146c3d00-916e-11eb-924e-4bedfa0afd59.png">

You were right @gtsiolis! First rendering of the `AllTeams` component is blocked by script loading.

Reply:
Smooth! Thanks for taking the time to verify this @AlexTugarev! 🙏 

If this is a quick fix, let's do it. Otherwise, I'd suggest to open a follow up issue to fix in the upcoming milestones (see https://github.com/gitpod-io/gitpod/pull/3656#discussion_r604028048). Your call! 📞

Reply:
perf issue is resolved ✔️

Reply:
Thank you @AlexTugarev! 🌟 🌟 🌟